### DISCUSSION TITLE
NanUI 1.0 代码仓促上线

### DISCUSSION THREAD

**Discussion by XuanchenLin** [XuanchenLin/NanUI#368] [公告 / Announcements]
没想到之前的公告引起了波澜，让平时冷冷清清的仓库有了些许热闹的氛围，感谢各位的支持！

今天跑了一天依然是没有任何收获的一天，中秋节前客户承诺中秋节后会有回款，但是现在国庆过完都几天了，还是毫无进展。明天估计还得继续去催款。

虽然略显仓促，但刚才我还是决定把新的 NanUI (WinFormium) 的代码 push 上来了。当前仓库 master 分支的代码没有示例，没有文档，有兴趣的朋友可以自行拉下来研究，明天晚上回来如果有时间，我尽量写个简单的演示新版基础接口的使用方法。至于文档和详细的接口示例只能后面再说了。

当然，如果有朋友愿意参与到 NanUI 的开发中，欢迎提交PR。

以上~

====== 10月10日分割线 ======

今天抽空写了个简单的示例，希望对各位对此项目感兴趣的朋友有所帮助。

Nuget包就先不发布了，因为不知道会不会有什么不可预测的问题，因为确实仓促。

剩下的时间，我会尽力去完成我目前应该完成工作，对公司和家人有所交代，之后我将全力完成 NanUI 正式版剩余的工作！

感谢各位的祝福和支持，借各位吉言，希望一切顺利，这几年不仅是开发这个行业不好做，各行各业都难，所以还是希望所有人到中年的朋友们都能生活稳定，家庭幸福吧！

未来可期，稍后再会！

**Comment by mengdodo** [comment]
先安心收款才是正事， good luck !

**Comment by hantsy** [comment]


**Reply by nellochen** [reply]
没看到承德的事情么，想干感觉只能出去了。

**Reply by crafet** [reply]
cool，大佬可以分享下吗

**Reply by whevether** [reply]
上一个承德的已经赃款充公了。

**Comment by zhangmeng8815** [comment]
牛人都转行了，不由的开始担心自己了。
祝好运

**Comment by mrtylerzhou** [comment]
这些年见证了一批批.net人员离开,从最初心里五味杂陈逐渐变得呆若木鸡毫无波澜.但是最近些天看到nanui作者选择离开,心里还是挺难受了.作为一个客户端偏执者,作为一个规模较大的快递公司客户端负责人,对nanui充满感情,是它让我在团队稳住了脚,是它让.net团队在公司有一席之地,对nanui的感情不是了了几句话能说清的.然后面包代替不了尊严,尊严也代替不了面包.大家都必须面对衣食住行这个基本问题.青山不改,绿水长流,江湖再见

**Comment by moonlightlang** [comment]
加油

**Comment by zk4** [comment]
我觉得你可以天天在这写程序员跨行日记，要火

**Reply by XuanchenLin** [reply]
都没时间和精力写代码了，哪还有能力还想着当网红呢😂😂

**Reply by just-dj** [reply]
是啊 前辈开个账号

**Comment by lantingBoy** [comment]
加油

**Comment by simba-cn** [comment]
加油！

**Comment by crafet** [comment]
respect!

**Comment by blueskytcg** [comment]
编码的工作很难找么？

**Comment by iLucky7** [comment]
respect!

**Comment by qizhendong1** [comment]
加油

**Comment by Mustenaka** [comment]
先活下来，这是最主要的

**Comment by clapat-bb** [comment]
加油加油
